### ISSUE TITLE
Visual indicator when a testnet is active

### ISSUE THREAD

**Issue by KPrasch** [nucypher/nuit#120]
Give the user a visual indication when an ethereum testnet is selected.  Perhaps this can be a badge next to the "Connect Wallet" button.

**Comment by derekpierre** [comment]
Should the network name also be displayed somewhere on the "Manage" page (perhaps others)?

**Comment by KPrasch** [comment]
Yes, not a bad idea.  This might fit well in the "Current Period" management panel.
